<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP принтера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIp"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Длина этикетки</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxWidth">
       <property name="maximum">
        <double>65535.000000000000000</double>
       </property>
       <property name="value">
        <double>110.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ширина этикетки</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxHeiht">
       <property name="maximum">
        <double>65535.000000000000000</double>
       </property>
       <property name="value">
        <double>95.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButtonCal">
       <property name="text">
        <string>Калибровать</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Зазор</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGap">
       <property name="maximum">
        <double>65535.000000000000000</double>
       </property>
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
